### PR TITLE
Invite Editor Overlay

### PR DESCRIPTION
**Pull request by jmar910**

### CODE REVIEW

**Comment by rizwanreza** [comment]
Looks good. :+1:

File: spec/support/pages/page.rb [hunk 1]
```diff
@@ -75,7 +75,6 @@ def view_card(card_name, overlay_class=nil, &block)
     retry_stale_element do
       find('.card-content', text: card_name).click
     end
-    synchronize_content! 'CLOSE'
```
Comment (mikem) [inline spec/support/pages/page.rb]: Why was this taken out? Was it causing an issue or just not necessary any longer? I think it serves to wait until the card is rendered before allowing the spec to continue.

Comment (nummi) [inline spec/support/pages/page.rb]: @mikem Line 84's `find()` also waits for the card to be rendered before continuing. In practice finding by css can also be quite a bit quicker than using `synchronise_content` on a big page.  All of our overlays have both 'close' button and the '.overlay' class, so checking for both felt redundant.

84: `overlay = overlay_class.new session.find(".overlay")`

Comment (mikem) [inline spec/support/pages/page.rb]: OK, makes sense. Thanks for clarifying :-)

**Pull request merged**